tests/Bridge/Symfony/Bundle/Command/SwaggerCommandTest.php

```diff
@@ -19,6 +19,7 @@
 
 /**
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ * @group legacy
```

Comment:
```
Remaining deprecation notices (220)

The use of ApiPlatform\Core\PathResolver\UnderscoreOperationPathResolver is deprecated since 2.1. Please use PathSegmentNameGenerator instead: 219x
    219x in SwaggerCommandTest::testExecute from ApiPlatform\Core\Tests\Bridge\Symfony\Bundle\Command

The "api_platform.operation_path_resolver.default" service is deprecated since ApiPlatform 2.1 and will be removed in 3.0. Use PathSegmentNameGenerator instead: 1x
    1x in SwaggerCommandTest::testExecute from ApiPlatform\Core\Tests\Bridge\Symfony\Bundle\Command

Legacy deprecation notices (62)
```

This makes phpunit exit with code `1`. We're still using `UnderscoreOperationPathResolver`, should I already remove the service use in the code? If I do so, it will lead to breaking changes. 

Or I can just change the `api_platform.operation_path_resolver.default` but then again, BC break.